app/components/CrashReportModal/CrashReportModal.component.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import type { FC } from 'react';
+
+import { Backdrop, Box, Button, Fade, makeStyles, Modal, Typography } from '@material-ui/core';
+import { useTranslation } from 'react-i18next';
+
+import type { Theme } from '../../theme';
+import debugLogger from '../../utils/debugLogger.client';
+import { LogoIcon } from '../icons';
+
+export interface CrashReportModalProps {
+  open: boolean;
+}
+
+const useStyles = makeStyles((theme: Theme) => {
+  return {
+    modal: {
+      alignItems: 'center',
+      display: 'flex',
+      justifyContent: 'center',
+    },
+    paper: {
+      backgroundColor: theme.palette.background.paper,
+      border: '2px solid #000',
+      boxShadow: theme.shadows[5],
+      padding: theme.spacing(2, 4, 3),
+    },
+    root: {},
+  };
+});
+
+const CrashReportModal: FC<CrashReportModalProps> = ({ open }: CrashReportModalProps) => {
+  const classes = useStyles();
+  const { t } = useTranslation('CrashReportModal');
+
+  const handleQuitApplication = () => {
+    debugLogger.crashApplication();
+  };
+
+  return (
+    <Modal
+      aria-labelledby="transition-modal-title"
+      aria-describedby="transition-modal-description"
+      className={classes.modal}
+      open={open}
+      closeAfterTransition
+      BackdropComponent={Backdrop}
+      BackdropProps={{
+        timeout: 1000,
+      }}
+      disableAutoFocus
+      disableEnforceFocus
+    >
+      <Fade in={open}>
+        <Box className={classes.paper} display="flex" flexDirection="column">
+          <LogoIcon />
+          <br />
+          <Typography color="textPrimary" gutterBottom variant="h2">
+            {t('uhOh')}
+          </Typography>
+          <br />
+          <Typography variant="body2" color="textSecondary">
+            {t('encounteredAnIssue')}
+          </Typography>
+          <br />
+          <Typography variant="body2" color="textSecondary">
+            {t('sendReport')}
+          </Typography>
+          <br />
+          <Box>
+            <Typography variant="body2" color="textSecondary" display="inline">
+              {t('sendDebugLog')}
+            </Typography>
+            <Button
+              color="primary"
+              onClick={() => {
+                debugLogger.openLogsFolder();
+              }}
```

Comment:
braces not needed

Reply:
So, from a readability standpoint I prefer to always use braces, even if it's a one liner. I find it helps tremendously in clarifying what is going on, especially since not every language supports this style of function writing. Thoughts on that at all?